ci/travis/install-dependencies.sh

```diff
@@ -202,6 +202,8 @@ install_toolchains() {
 install_dependencies() {
 
   install_bazel
+  "${WORKSPACE_DIR}"/ci/suppress_output bazel fetch --noshow_progress --color=no "//:*" &  # Begin fetching packages in the background (network-bound)
```

Comment:
will this lock the entire bazel workspace? if we do a build later on, will it ```
Another command (pid=19993) is running.  Waiting for it to complete on the server...
```

Reply:
Yup. It needs to wait for fetching to complete one time or another so that should work fine.

Reply:
Fetching in CI is relatively fast (running data centers), how much time does this really save us? The added complexity might not worth it.

Reply:
I'll see if I can measure, it's hard but if I can't show an improvement we can take it out.

Reply:
I tried doing a dry-run (`bazel build --check_up_to_date` in lieu of `install_ray`, and then exiting) but the CI performance appears to be very inconsistent different OSes and different runs. Here's one example across OSes:
```
Windows: 21:48 -> 19:54 (saved 1:54)
Ubuntu:   6:36 ->  6:49 (lost  0:13)
Mac:      9:41 -> 11:20 (lost  1:39)
```
It's difficult to interpret, but here are my takeaways:
- I think it is likely that it genuinely does differ across OSes, due to I/O characteristics
- The slowdown on Mac is possibly due to CI inconsistency (but I can re-run to check?)
- The slowdown on Ubuntu is potentially genuine, though it seems negligible in any case
- The speedup on Windows is likely genuine (and the most important I think, since people don't run Windows locally), but the true magnitude may be different (e.g. 1 min instead of 2 mins), I'd have to re-run to see.